### PR TITLE
Feature/with models

### PR DESCRIPTION
**Pull request by StuWebDev**
Wow... where to start.

Jira Card [DIGITAL-8178](https://stockportbi.atlassian.net/browse/DIGITAL-8178) is about moving the models package into the gateways, and deprecating the models repo/package/etc.

This is step one, the migration of the models package into the Models folder of the Gateways package.

Even though the card said to maintain the namespace of models, I think for future development it would be easier then to do the harder work now. So renamed the namespaces to reflect the current structure.

Also added in a JsonConverter for System.Text.Json instead of Newtonsoft, which I used in the new WiredPlus models/gateway

Upgraded the dependencies and did a major version increment of this package.

### CODE REVIEW

File: src/GovUKPay/Models/Response/NextUrlPost.cs [hunk 1]
```diff
@@ -3,8 +3,8 @@ namespace StockportGovUK.NetStandard.Gateways.GovUKPay.Models.Response
     public class NextUrlPost
     {
         public string type { get; set; }
-        
-        public Params @params { get; set; }     
+
+        public Params @params { get; set; }
         public string href { get; set; }
```
Comment (lydiagray) [inline src/GovUKPay/Models/Response/NextUrlPost.cs:8]: Should we have an empty line between these 2?

File: src/Netcall/ConverseGateway.cs [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
 using StockportGovUK.NetStandard.Gateways.Netcall.Models;
 
 namespace StockportGovUK.NetStandard.Gateways.Netcall
 {
     public class ConverseGateway : Gateway, IConverseGateway
     {
-        
+
```
Comment (lydiagray) [inline src/Netcall/ConverseGateway.cs:9]: Do we need the empty line here?

File: src/Netcall/NetcallGateway.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@ namespace StockportGovUK.NetStandard.Gateways.Netcall
 {
     public class NetcallGateway : Gateway, INetcallGateway
     {
-        
+
```
Comment (lydiagray) [inline src/Netcall/NetcallGateway.cs:8]: Extra line?

File: src/Uniform/IUniformServiceGateway.cs [hunk 4]
```diff
@@ -1,6 +1,6 @@
-﻿using StockportGovUK.NetStandard.Gateways.Response;
-using System.Threading.Tasks;
-using StockportGovUK.NetStandard.Models.Uniform;
+﻿using System.Threading.Tasks;
+using StockportGovUK.NetStandard.Gateways.Models.Uniform;
+using StockportGovUK.NetStandard.Gateways.Response;
```
Comment (lydiagray) [inline src/Uniform/IUniformServiceGateway.cs:5]: Extra line?

File: src/JsonConverters/DateTimeConverters.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+using System.Text.Json;
+using System;
+
+namespace StockportGovUK.NetStandard.Gateways.JsonConverters
+{
+    public class DateTimeConverters
+    {
+        private const string frenchCanadaLocale = "yyyy-MM-dd HH:mm:ss";
+
+
```
Comment (lydiagray) [inline src/JsonConverters/DateTimeConverters.cs:12]: Extra line?

**Review by jonhadley — APPROVED** [review]

**Pull request merged**